e2e-polybft/e2e/allowlist_test.go

```diff
@@ -107,3 +107,97 @@ func TestAllowList_ContractDeployment(t *testing.T) {
 		expectRole(types.ZeroAddress, allowlist.NoRole)
 	}
 }
+
+func TestAllowList_Transactions(t *testing.T) {
+	// create two accounts, one for an admin sender and a second
+	// one for a non-enabled account that will switch on-off between
+	// both enabled and non-enabled roles.
+	admin, _ := wallet.GenerateKey()
+	target, _ := wallet.GenerateKey()
+	other, _ := wallet.GenerateKey()
+
+	adminAddr := types.Address(admin.Address())
+	targetAddr := types.Address(target.Address())
+	otherAddr := types.Address(other.Address())
+
+	cluster := framework.NewTestCluster(t, 3,
+		framework.WithPremine(adminAddr, targetAddr, otherAddr),
+		framework.WithTransactionsAllowListAdmin(adminAddr),
+		framework.WithTransactionsAllowListEnabled(otherAddr),
+	)
+	defer cluster.Stop()
+
+	cluster.WaitForReady(t)
+
+	// bytecode for an empty smart contract
+	bytecode, _ := hex.DecodeString("6080604052348015600f57600080fd5b50603e80601d6000396000f3fe6080604052600080fdfea265627a7a7231582027748e4afe5ee282a786005d286f4427f13dac1b62e03f9aed311c2db7e8245364736f6c63430005110032")
+
+	expectRole := func(addr types.Address, role allowlist.Role) {
+		out := cluster.Call(t, contracts.AllowListTransactionsAddr, allowlist.ReadAllowListFunc, addr)
+
+		num, ok := out["0"].(*big.Int)
+		if !ok {
```

Comment:
Nitpick: `require.True(ok)` or `assert.True(ok)`

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1349/commits/3692badaa4e9300c8691a88752fe68c72114db65